app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  
+  def new
```

Comment:
We are overriding all of `devise` 's methods here. Please refer to [this](https://github.com/plataformatec/devise#configuring-controllers) part of the `devise` documentation for details on configuring its controllers. When you override `devise` 's default actions, you are foregoing the benefits it provides you.

If fully overriding the registrations_controller is what you really want then keep this. But just confirm with the documentation about it first.